src/oidentd.js

```diff
@@ -1,68 +1,42 @@
 "use strict";
 
-var fs = require("fs");
-var Helper = require("./helper");
+const fs = require("fs");
+const Helper = require("./helper");
 
 function OidentdFile(file) {
 	this.file = Helper.expandHome(file);
 	this.connectionId = 0;
-	this.connections = {};
+	this.connections = new Map();
 
 	this.refresh();
 }
 
 OidentdFile.prototype = {
-	hookSocket: function(socket, user) {
-		var that = this;
-		var id = null;
+	addSocket: function OidentdFileAddSocket(socket, user) {
```

Comment:
If you make `OidentdFile` a class, I believe you could write these something like:

```es6
class OidentdFile {
  /* ... */

  addSocket(socket, user) {
    /* ... */
  }

  /* ... */
}
```

I definitely don't like naming a method differently than its descriptor (`addSocket` vs. `OidentdFileAddSocket`) and I even dislike more naming non-class-like functions with an uppercase, so I think that would be a win-win :)